### ISSUE TITLE
code clarification

### ISSUE THREAD

**Issue by taylor-bsg** [black-parrot/black-parrot#380]
https://github.com/black-parrot/black-parrot/blob/9b3cea5c453d307f893044be6cd5c09f907cab9e/bp_be/src/include/bp_be_dcache/bp_be_dcache_pkt.vh#L9

if this structure is used just for mmu-dcache interaction, perhaps it should have tlb or mmu in the name?

**Comment by dpetrisko** [comment]
It's actually the part of the dcache request that _doesn't_ come from the TLB.  The comment should be changed

**Comment by taylor-bsg** [comment]
Yeah I realized; I generally think of MMU to mean TLB but here it means ld/st unit.

**Comment by dpetrisko** [comment]
Yep! All the names around that area could use some work
